DESCRIPTION

```diff
@@ -18,7 +18,9 @@ Imports:
     optparse,
     R6,
     stringr,
-    tools
+    tools,
+    here,
```

Comment:
1. keep dependencies in alphabetical order
2. for `here` specifically, it seems this is only used in tests. That means it should go under "Suggests" (for dev dependencies; see https://r-pkgs.org/dependencies-mindset-background.html#sec-dependencies-imports-vs-suggests)